lib/compat/wordpress-6.0/class-wp-theme-json-6-0.php

```diff
@@ -353,7 +355,11 @@ protected function get_block_classes( $style_nodes ) {
 			}
 
 			// 2. Generate the rules that use the general selector.
-			$block_rules .= static::to_ruleset( $selector, $declarations );
+			$styles = gutenberg_style_engine_generate( $node, array( 'selector' => $selector ) );
+			if ( isset( $styles['css'] ) ) {
+				$block_rules .= $styles['css'];
+			}
+
```

Comment:
Just testing for now. Ignore the compat location 😄 

The style engine doesn't take into account whatever is happening to duotone declarations above.